src/Components/Form/FormFields/Autocomplete.tsx

```diff
@@ -136,6 +138,21 @@ export const Autocomplete = <T, V>(props: AutocompleteProps<T, V>) => {
       ? options.filter((o) => o.search.includes(query))
       : options;
 
+  const prevQuery = useRef("");
+  useEffect(() => {
+    if (
+      query &&
+      prevQuery.current !== query &&
+      filteredOptions.length === 0 &&
+      props.noResultsLabel
+    ) {
+      Notifications.Error({
+        msg: props.noResultsLabel,
+      });
+    }
+    prevQuery.current = query;
+  }, [query, filteredOptions, props.noResultsLabel]);
+
```

Comment:
Instead of making the input component dispatching the notification (which might affect ALL other places), let's keep it where the user details are fetched (the network query)

```suggestion
```